src/components/ResourceHeader/_resource-header.scss

```diff
@@ -0,0 +1,123 @@
+.bx--resource-header {
```

Comment:
Should we leverage the `prefix` and `exports` stuff from `carbon-components` to help out with styles here?